### PR TITLE
Added missing fields to 'icinga_host' and 'icinga_host_template'

### PR DESCRIPTION
**Pull request by HeikoNeblung**
This PR will add all the fields mentioned in #59 by @rndmh3ro

Added mising properties:

* enable_notifications
* enable_active_checks
* enable_passive_checks
* enable_event_handler
* enable_flapping
* enable_perfdata
* volatile
* check_period
* check_timeout
* flapping_threshold_high
* flapping_threshold_low
* icon_image
* icon_image_alt

to modules 'icinga_host' and 'icinga_host_template'

### CODE REVIEW

File: plugins/modules/icinga_host.py [hunk 1]
```diff
@@ -215,6 +288,19 @@ def main():
...
+        enable_flapping=dict(type="bool", choices=[True, False], required=False),
+        enable_perfdata=dict(type="bool", choices=[True, False], required=False),
+        volatile=dict(type="bool", choices=[True, False], required=False),
+        check_period=dict(type="str", required=False),
+        check_timeout=dict(type="str", required=False),
+        flapping_threshold_high=dict(type="str", required=False),
+        flapping_threshold_low=dict(type="str", required=False),
+        icon_image=dict(type="str", required=False),
+        icon_image_alt=dict(type="str", required=False),
     )
```
Comment (rndmh3ro) [inline plugins/modules/icinga_host.py:304]: These are missing here (and below and also in the host template):
```
max_check_attempts: 3
retry_interval: "1m"
```

**Review by rndmh3ro — CHANGES_REQUESTED** [review]
The integration tests fail, because two parameters are missing. You need to make sure that a new parameter is at four places in the module:
- docs
- examples
- twice in the code

**Comment by rndmh3ro** [comment]
Also the linting fails. You can run this manually after installing tox by entering: `tox -elinters`

File: plugins/modules/icinga_host.py (hunk 1 above)
Comment (HeikoNeblung) [reply plugins/modules/icinga_host.py:366]: I added the two lines and hope you don't mind reordering them. I think it makes the code more maintainable.

**Comment by rndmh3ro** [comment]
After https://github.com/T-Systems-MMS/ansible-collection-icinga-director/pull/159 is merged, can you please rebase your PR on master? Then all tests will hopefully pass.

Also please try to separate formatting changes from code-changes next time. :) Otherwise its hard to see what you added and what only changed in the sorting..

**Comment by HeikoNeblung** [comment]
Sorry for mixing things up. I just wanted to keep the history clean.
Maybe the `tox -elinters` run is a thing to put in CONTRIBUTION.md?

**Comment by rndmh3ro** [comment]
Thanks for this PR!

>  Maybe the tox -elinters run is a thing to put in CONTRIBUTION.md?

Thanks here, too. I updated it with the information from the readme:
https://github.com/T-Systems-MMS/ansible-collection-icinga-director/blob/master/CONTRIBUTING.md

**Pull request merged**